x-pack/plugins/osquery/public/packs/types.ts

```diff
@@ -16,6 +16,7 @@ export interface IQueryPayload {
 export type PackSavedObject = SavedObject<{
   name: string;
   description: string | undefined;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   queries: Array<Record<string, any>>;
```

Comment:
Will the `unknown` work here instead of `any`?

Reply:
should work, thanks!